docs/content/manual/pre-release/node-not-ready/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Node Not Ready
+---
+Consolidate test cases for [#5569 [TEST] node down/reboot and kubelet restart](
+https://github.com/longhorn/longhorn/issues/5569)
```

Comment:
I missed this before merging.
@roger-ryao can you please remove these lines, it's not needed?

Reply:
> I missed this before merging. @roger-ryao can you please remove these lines, it's not needed?

Hi @khushboo-rancher 
      [PR-1343](https://github.com/longhorn/longhorn-tests/pull/1343) is ready for review.